components/PowerInfoCard.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+import Image from 'next/image';
+
+type PowerInfo = {
+  src: string;
+  applicance: string;
+  kWh: string;
+};
+
+const PowerInfoCard: FC<PowerInfo> = ({ src, applicance, kWh }) => {
```

Comment:
appliance is spelled wrong here 
```suggestion
const PowerInfoCard: FC<PowerInfo> = ({ src, appliance, kWh }) => {
```